server/src/main/scala/io/delta/standalone/internal/model.scala

```diff
@@ -16,12 +16,40 @@
 
 package io.delta.standalone.internal
 
+import com.fasterxml.jackson.databind.annotation.JsonDeserialize
 import io.delta.standalone.internal.actions.{
+  Format => DeltaFormat,
   Metadata => DeltaMetadata,
   Protocol => DeltaProtocol,
   SingleAction => DeltaSingleAction
 }
 
+case class DeltaMetadataCopy(
```

Comment:
Consider adding a comment header on why this is needed.

Reply:
added